tinygrad/tensor.py

```diff
@@ -14,7 +14,7 @@ def require_init_gpu():
   if cl_queue is None:
     try:
       # for Macbook 16 inch
-      cl_ctx = cl.create_some_context(answers=[0,2])
```

Comment:
Why did this change?

Reply:
Needed this to work on an older Mac, didn't mean to push this.